docs/ui/components/Footer/Links.tsx

```diff
@@ -0,0 +1,93 @@
+import { css } from '@emotion/react';
+import {
+  CodeIcon,
+  EditIcon,
+  GithubIcon,
+  iconSize,
+  MessageIcon,
+  spacing,
+  theme,
+  typography,
+} from '@expo/styleguide';
+
+import { A, LI } from '../Text';
+import { githubUrl } from './utils';
+
+export const IssuesLink = ({ title }: { title: string }) => (
+  <LI>
+    <A css={linkStyle} target="_blank" href={`https://github.com/expo/expo/labels/${title}`}>
```

Comment:
```suggestion
    <A css={linkStyle} openInNewTab href={`https://github.com/expo/expo/labels/${title}`}>
```
This should open the link in a new tab with the appropriate attributes.